calico_kubernetes/calico_cni.py

```diff
@@ -0,0 +1,422 @@
+# Copyright 2015 Metaswitch Networks
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+import socket
+import functools
+import logging
+import json
+import os
+import sys
+
+from subprocess import check_output, CalledProcessError, check_call
+from netaddr import IPAddress, IPNetwork, AddrFormatError
+from docker import Client
+from docker.errors import APIError
+import pycalico
+from pycalico.netns import PidNamespace,remove_veth
+from pycalico.ipam import IPAMClient, SequentialAssignment
+from pycalico.datastore_datatypes import Rules, IPPool
+from logutils import configure_logger
+
+ETCD_AUTHORITY_ENV = 'ETCD_AUTHORITY'
+LOG_DIR = '/var/log/calico/calico-k8'
```

Comment:
LOG_DIR should be "/var/log/calico/kubernetes" (like it is currently).

The LOGFILE_NAME can be "calico-kubernetes-cni.log" to distinguish it from the existing log.